rgsl-nlinear/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
Why did you create this new crate?

Reply:
I wasn't sure if you wanted rust-GSL to be pure Rust or not.  It could easily be combined with the Rust-GSL crate.

Reply:
I want it to remain pure Rust indeed. But then, why do you need this C code?

Reply:
So I'm posting a link to an issue comment that I submitted a few weeks ago:

https://github.com/GuillaumeGomez/rust-GSL/issues/68#issuecomment-2409928724

I had trouble setting the default parameters in Rust using just the GSL-sys library.  All the function does is return a struct, but the struct, written in C, returns other substructs, some of which contain references to C functions.  I think this is what is causing Bus errors and Segmentation faults when setting the default parameters.

Reply:
Would be better to fix the issue rather than trying to rewrite some code in C.

Reply:
Agreed.  However, I only have about a month of Rust under my belt.  I just got the rgsl-nlinear crate to work how I would expect it to work, and I have the feeling that this issue is beyond my current level of expertise, but I have some ideas on a way forward.

Reply:
I figured out a workaround to the issue that I was experiencing and have rewritten the crate in Rust.  Please let me know if I can resolve this conversation.

Reply:
It looks much better. But I'm still not sure why it's creating a new crate.